tools/cli/adminDBScanCommand.go

```diff
@@ -157,3 +163,137 @@ func checkExecution(
 
 	return execution, invariant.NewInvariantManager(ivs).RunChecks(ctx, execution)
 }
+
+func AdminDBScanUnsupportedWorkflow(c *cli.Context) {
+	numberOfShards := c.Int(FlagNumberOfShards)
+	rps := c.Int(FlagRPS)
+	outputFile := getOutputFile(c.String(FlagOutputFilename))
+	defer outputFile.Close()
+	for i := 0; i < numberOfShards; i++ {
+		listExecutionsByShardID(c, i, rps, outputFile)
```

Comment:
Since the script may be long running, I will we should provide user with some progress information, like shard xxx has completed.

We can also consider give user the option to start from middle in case the scan file due to some transient error. Probably add flags for specifying shardID range instead of just the max shardID?

Reply:
Changed to use lower shard boundary and upper shard boundary.